File: Modul-3/Soal-2/PRAK302-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {

    int bil;
    printf("Masukan angka: ");
    scanf("%d", &bil);

    if (bil >= 80) {
        printf("nilai huruf adalah A\n");
    }
    else if (bil >= 70 && bil < 80) {
        printf("nilai huruf adalah B\n");
    }
    else if (bil >= 60 && bil < 70) {
        printf("nilai huruf adalah C\n");
    }
    else if (bil >= 50 && bil < 60) {
        printf("nilai huruf adalah D\n");
    }
    else {
        printf("nilai huruf adalah E\n");
    }

    return 0;
}

File: Modul-3/Soal-2/PRAK302-2410817320001-NazlaSalsabila.py
bil = int(input("Masukan angka: "))

if bil >= 80:
    print("nilai huruf adalah A")
elif bil >= 70 and bil < 80:
    print("nilai huruf adalah B")
elif bil >= 60 and bil < 70:
    print("nilai huruf adalah C")
elif bil >= 50 and bil < 60:
    print("nilai huruf adalah D")
else:
    print("nilai huruf adalah E")